### ISSUE TITLE
Support anchor property for Image and ImageRGBA

### ISSUE THREAD

**Issue by bryevdv** [bokeh/bokeh#1763]
I thought we already did but apparently not:

```
bryan@laptop (git:task/1747_model_docs) ~/work/bokeh/bokehjs/src/coffee/renderer/glyph $ grin anchor
./image_url.coffee:
   52 :       anchor = @mget('anchor') or "top_left"
   54 :       switch anchor
```

**Comment by bryevdv** [comment]
Discovered in #1747

**Comment by jakirkham** [comment]
Could this offer a fix to this ( #1666 ) or is it unrelated?

**Comment by jussyr** [comment]
Any chance of having the anchor option added to image ( #1666 )?  If this is in the image_url, how difficult would it be to port to image?
